src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -169,6 +224,25 @@ class EmojiPickerMenu extends Component {
                     getItemLayout={this.getItemLayout}
                     showsVerticalScrollIndicator
                 />
+                {!!this.state.value && (
```

Comment:
NAB I think `Boolean(!this.state.value)` is more common in the code base

Reply:
👍🏻